context/app/static/js/pages/Cells/Cells.jsx

```diff
@@ -35,28 +35,30 @@ function Cells() {
         given="a list of genes, a minimum expression level, and a minimum percentage of cells at that expression level"
         returns="a list of UUIDs for datasets which meet those minimums"
       />
-      <StepAccordion
-        summaryHeading="Parameters"
-        content={
-          <DatasetsSelectedByExpression
-            setResults={setResults}
-            minExpressionLog={minExpressionLog}
-            setMinExpressionLog={setMinExpressionLog}
-            minCellPercentage={minCellPercentage}
-            setMinCellPercentage={setMinCellPercentage}
-            geneNames={geneNames}
-            setGeneNames={setGeneNames}
-          />
-        }
-      />
-      <StepAccordion
-        summaryHeading="Results"
-        disabled={results.length === 0}
-        content={
-          results.length > 0 ? (
-            <DatasetsTable datasets={results} minGeneExpression={10 ** minExpressionLog} geneName={geneNames[0]} />
-          ) : undefined
-        }
+      <AccordionSteps
+        steps={[
+          {
+            heading: 'Parameters',
+            content: (
+              <DatasetsSelectedByExpression
+                setResults={setResults}
+                minExpressionLog={minExpressionLog}
+                setMinExpressionLog={setMinExpressionLog}
+                minCellPercentage={minCellPercentage}
+                setMinCellPercentage={setMinCellPercentage}
+                geneNames={geneNames}
+                setGeneNames={setGeneNames}
+              />
+            ),
+          },
+          {
+            heading: 'Results',
+            content:
+              results.length > 0 ? (
+                <DatasetsTable datasets={results} minGeneExpression={10 ** minExpressionLog} geneName={geneNames[0]} />
+              ) : undefined,
+          },
+        ]}
       />
```

Comment:
I'll keep it as is for now since it's non-essential and I have some downstream branches, but I'll give it some thought and revisit. I went in this direction because it avoids having to handle the situation of worrying about the length of the headings array matching the length of the children.